www/js/lib/zimfile.js

```diff
@@ -110,6 +110,42 @@ define(['xzdec_wrapper', 'util', 'utf8', 'q', 'zimDirEntry'], function(xz, util,
         }
     };
 
+    /**
+     * Reads the whole MIME type list and returns it as a populated Map
+     * The mimeTypeMap is extracted once after the user has picked the ZIM file
+     * and is stored as ZIMFile.mimeTypes
+     * 
+     * @returns {Promise} A promise for the MIME Type list as a Map
+     */
+    ZIMFile.prototype._mimeTypeMap = function() {
```

Comment:
This should be renamed with an action verb, as it's not a simple attribute. Something like `readMimeTypesMap`, for example

Reply:
OK